src/reporters/test-case.reporter.js

```diff
@@ -1,5 +1,6 @@
 const WDIOReporter = require("@wdio/reporter").default
 const colors = require("colors");
+const {TraceabilityUtility} = require("../../lib/utils/traceability.util");
```

Comment:
Can barrels be used in this context? (just asking, as I just have used them for imports)